src/containers/single/breaking-changes.tsx

```diff
@@ -254,16 +190,7 @@ export default function render() {
               We will notify you when breaking changes are announced, and send
               periodic reminders before they take effect.
             </SubscribeP>
-            <MailchimpSubscribe
-              url={url}
-              render={({ subscribe, status, message }) => (
-                <CustomForm
-                  status={status}
-                  message={message}
-                  onValidated={(formData) => subscribe(formData)}
-                />
-              )}
-            />
+            <MarketoForm />
```

Comment:
If you generalize this could be:

```suggestion
            <MarketoForm url="//info.saasquatch.com" munchkinId="162-BJJ-156" formid=1018 />
```

And it pushes the usage code of this specific form into the place that specifically needs it.